e2e/components/OverflowMenu/OverflowMenu-test.avt.e2e.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright IBM Corp. 2016, 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+const { expect, test } = require('@playwright/test');
+const { visitStory } = require('../../test-utils/storybook');
+
+test.describe('OverflowMenu', () => {
+  test('accessibility-checker @avt', async ({ page }) => {
+    await visitStory(page, {
+      component: 'OverflowMenu',
+      id: 'components-overflowmenu--default',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('OverflowMenu');
+  });
+
+  test('accessibility-checker render custom icon @avt', async ({ page }) => {
+    await visitStory(page, {
+      component: 'OverflowMenu',
+      id: 'components-overflowmenu--render-custom-icon',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('OverflowMenu-custom-icon');
+  });
+
+  test('accessibility-checker-menu-open @avt', async ({ page }) => {
+    await visitStory(page, {
+      component: 'OverflowMenu',
+      id: 'components-overflowmenu--default',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page.getByRole('button')).toBeVisible();
+
+    // Tab and open the toggle button for the OverflowMenu
+    await page.keyboard.press('Tab');
+    const toggleButton = page.getByRole('button');
+    await expect(toggleButton).toBeFocused();
+    await page.keyboard.press('Enter');
+    await expect(toggleButton).toHaveClass(/cds--overflow-menu--open/);
+
+    await expect(page).toHaveNoACViolations('OverflowMenu-open');
+  });
+
+  test('overflow-menu - keyboard nav', async ({ page }) => {
+    await visitStory(page, {
+      component: 'OverflowMenu',
+      id: 'components-overflowmenu--default',
+      globals: {
+        theme: 'white',
+      },
+    });
+    await expect(page.getByRole('button')).toBeVisible();
+
+    // Tab and open the toggle button for the OverflowMenu
+    await page.keyboard.press('Tab');
+    const toggleButton = page.getByRole('button');
+    await expect(toggleButton).toBeFocused();
+    await page.keyboard.press('Enter');
+    await expect(toggleButton).toHaveClass(/cds--overflow-menu--open/);
+    // Navigation inside the menu
+    await expect(page.getByTestId('stop-app')).toBeFocused();
+    await page.keyboard.press('ArrowDown');
+    await expect(page.getByTestId('restart-app')).toBeFocused();
```

Comment:
Could we get these instead via role like this: `page.getByRole('button', {name: 'Stop app'})`? This way we could remove the reliance on the testid in the story

Reply:
Hey @tay1orjones 
It doesn't work. Probably because it has a `div` inside the button.

```
<button class="cds--overflow-menu-options__btn" role="menuitem" tabindex="-1" data-testid="stop-app">
    <div class="cds--overflow-menu-options__option-content">Stop app</div>
</button>
```
 
I also tried another ways to get this button for example: 
`await expect(page.getByText('Stop app')).toBeFocused();`

But the only way it works was with the data-testid. And I agree with you, it doesn't look good to have that `data-testid` in the story... 
I'm gonna try to spend a little bit more time on it, to see if I find a solution that doesn't rely on using `data`testid`